### DISCUSSION TITLE
FlaxDog

### DISCUSSION THREAD

**Discussion by Yamanipanuchi** [guydavis/machinaris#167] [Q&A] [answered]
I added Discord and Push to FlaxDog config and enabled the notifications but I am not getting the notifications. I've done over all the configs that I can see but I haven't got a notification.

Is there another setting I need to change to enable it?

**Comment by guydavis** [comment, marked as answer]
Hi! Please verify that:
1.  Your Flax log is set to INFO level.  Simply restarting the Machinaris container will ensure this.  Go to Farming page, chose your Worker and View Flax Logs, you should see verbose output every few seconds.
2. That FlaxDog is running and logging: Go to the Alerts page, select your Worker, and chose 'View Flax Monitoring Logs'.

Look for any errors in above.  Please review the [wiki here](https://github.com/guydavis/machinaris/wiki/ChiaDog).   If still not working, please attach your Flaxdog configuration to this issue, from the Settings | Alerts page.  Hope this helps!

**Comment by Yamanipanuchi** [comment]
Okey, Flax is logging at INFO level but Flaxdog does not seem to see it... So I would guess Flaxdog isn't seeing the log files... Checking the config.
 Ill also re-start the docker to make sure.

**Comment by Yamanipanuchi** [comment]
It looks as if the Re-start worked. I realized since I did the update that included FLAX I have not, And im going to guess FLAXDOG did not see the logs initially because they didn't exist yet.
